### PR TITLE
ISSUE-397 Auto-complete contact integration test for webadmin

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraContactAutocompleteMethodContract.scala [hunk 1]
```diff
@@ -803,4 +862,57 @@ trait LinagoraContactAutocompleteMethodContract {
...
+                      |  "sessionState": "${SESSION_STATE.value}",
+                      |  "methodResponses": [
+                      |    [
+                      |     "TMailContact/autocomplete",
+                      |		  {
+                      |			  "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+                      |			  "list": [],
+                      |       "limit": 256
+                      |		  },
+                      |		  "c1"
```
Comment (quantranhong1999) [inline tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraContactAutocompleteMethodContract.scala:912]: Please fix indent here.

**Review by quantranhong1999 — APPROVED** [review]

**Comment by Arsnael** [comment]
Rebase on master

**Comment by quantranhong1999** [comment]
Could you add an IT that reproduces the issue I raised [here](https://github.com/linagora/tmail-backend/pull/407)?. 
Feel free to cherry-pick or suggest me a better route path.

**Review by vttranlina — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
> Could you add an IT that reproduces the issue I raised [here](https://github.com/linagora/tmail-backend/pull/407)?. Feel free to cherry-pick or suggest me a better route path.

Done :)

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**